docker_test.go

```diff
@@ -83,3 +83,31 @@ func TestPortMappingLabels(t *testing.T) {
 	assert.NotNil(t, pm.Labels)
 	assert.Equal(t, 0, len(*pm.Labels))
 }
+
+func TestSimpleVolumes(t *testing.T) {
```

Comment:
I think we can drop this one if we decide to narrow `ExternalVolume`'s purpose as suggested elsewhere.